src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionSpec.scala

```diff
@@ -38,11 +38,36 @@ import org.scalatest.matchers.should.Matchers
 import org.scalatest.{BeforeAndAfterAll, PrivateMethodTester}
 import org.scalatestplus.mockito.MockitoSugar
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.ExecutionContext.Implicits.{global => globalEc}
 import scala.concurrent.Future
 import scala.concurrent.duration._
 import scala.util.{Success, Try}
 
+class SamMockGoogleDirectoryDAO extends MockGoogleDirectoryDAO {
```

Comment:
This exists because at some point the MockGoogleDirectoryDAO lost the lower method. The strictly correct fix would be to go into workbench libs and make the change there. I want to follow up with that and then go back and change this, because I want to move as fast on the java upgrade as possible, since its chance to go stale is very high.

Reply:
https://broadworkbench.atlassian.net/browse/CA-1803